### ISSUE TITLE
Get pods via LabelSelctor

### ISSUE THREAD

**Issue by polonel** [godaddy/kubernetes-client#144]
When trying to get pods via label selector I get all pods returned in the name space. 
`core.ns.po.get({ qs: { labelSelector: 'mylabel=thelabel' } }, (err, results) = > { console.log(results); });`

Not sure what the correct method would be for getting pods based on label selectors. Looking through the code base, I noticed the .get on pods will not accept the label selector.
https://github.com/godaddy/kubernetes-client/blob/master/lib/replicationcontrollers.js#L46

Am I overlooking something simple?

**Comment by tkporter** [comment]
Hey @polonel, something like this should work:
```
core.ns.po.matchLabels({ mylabel: 'thelabel' }).get((err, result) => {
  console.log(JSON.stringify(result));
});
```

**Comment by polonel** [comment]
Thanks for clarify this.

**Comment by frostebite** [comment]
This also seems to not work? No docs on query selector? An example would be great @silasbw

**Comment by silasbw** [comment]
Yeah, the approach @tkporter mentions is for a previous version of the kubernetes-client API.  I'll re-open this with the intention of adding an example.

**Comment by frostebite** [comment]
I'm happy to create an example and PR it into the docs but I'm still needing a simple valid example :)

**Comment by kkolk** [comment]
This works for me:

`let pods = await client.api.v1.pods.get({ qs: { labelSelector: "app=MyTestApp"}});`
